ods_ci/tests/Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource

```diff
@@ -329,7 +335,9 @@ Check Distributed Worklaod Status Overview
     Check Expected String Contains    //td[@data-label="Name"]    ${job_name}
     Check Expected String Equals    //td[@data-label="Priority"]    0
     ${get_date} =    Get Current Date    result_format=%-m/%-d/%Y
-    Check Expected String Contains    //td[@data-label="Created"]    ${get_date}
+    ${get_created_datetime} =    Get Text  xpath=//td[@data-label="Created"]
+    ${ui_parsed_date} =    Evaluate    re.search(r'[^,]+','''${get_created_datetime}''').group(0)
```

Comment:
Please be consistent with the variable name.
Suggestion: Rename `${ui_parsed_date}` it to `get_created_date`